core/network/src/service.rs

```diff
@@ -531,8 +531,14 @@ fn run_thread<B: BlockT + 'static>(
 						info!(target: "sync", "Banning {:?} because {:?}", who, message);
 						network_service_2.lock().ban_node(who)
 					},
-					Severity::Useless(_) => network_service_2.lock().drop_node(who),
-					Severity::Timeout => network_service_2.lock().drop_node(who),
+					Severity::Useless(message) => {
+						info!(target: "sync", "Dropping {:?} because {:?}", who, message);
```

Comment:
You probably wanted `Dropping {:?} because it is useless` here.

Reply:
@tomaka this message is correct as is?

Reply:
We want to print the message that comes with the `Useless`, and the context is probably enough to understand the message.